src/core/Engine.js

```diff
@@ -322,21 +318,16 @@ var Body = require('../body/Body');
      * @method _bodiesUpdate
      * @private
      * @param {body[]} bodies
-     * @param {number} deltaTime 
-     * The amount of time elapsed between updates
-     * @param {number} timeScale
-     * @param {number} correction 
-     * The Verlet correction factor (deltaTime / lastDeltaTime)
-     * @param {bounds} worldBounds
+     * @param {number} delta The amount of time elapsed between updates
      */
-    Engine._bodiesUpdate = function(bodies, deltaTime, timeScale, correction, worldBounds) {
+    Engine._bodiesUpdate = function(bodies, delta) {
         for (var i = 0; i < bodies.length; i++) {
             var body = bodies[i];
 
             if (body.isStatic || body.isSleeping)
```

Comment:
The deltaTime of the static and sleeping bodies do not get set. So if you are doing `getVelocity` on them you'll get `NaN`.

Reply:
Good point, I think this should be fixed now with the initial value for `body.deltaTime`?